lib/sles4sap.pm

```diff
@@ -1452,6 +1454,28 @@ sub modify_selinux_setenforce {
     }
 }
 
+=head2 get_test_repos
+
+    Constructs an array of repos to be added to SUTs.
+
+=cut
+
+sub get_test_repos {
+    # In Incidents there is INCIDENT_REPO instead of MAINT_TEST_REPO
+    # Those two variables contain list of repositories separated by comma
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
```

Comment:
```
set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
```

I'd like (not in this PR) try to remove this `set_var` and maybe this line at all. At least I do not like `set_var` in general, and I like even less if it is "buried" within a lib

Reply:
It would be easy to directly grab the repos from either INCIDENT_REPO or MAINT_TEST_REPO. What I'm not sure abut is wheher we actually *need* to populate MAINT_TEST_REPO for some other reason.

Reply:
I found 32 instances where `MAINT_TEST_REPO` is referenced without `set_var()` in os-autoinst-distri-opensuse, so I agree with @BillAnastasiadis about setting it in case it's needed later.